packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -157,6 +157,13 @@ export class Viewer {
    */
   @Prop() public keyboardControls = true;
 
+  /**
+   * Enables or disables the default rotation interaction being changed to
+   * rotate around the mouse down location. This requires the enabling of
+   * depth buffers through the viewer's `streamAttributes`.
```

Comment:
Can do this in a follow up, but IMO enabling this prop should enable depth buffers automatically.